### PR TITLE
fix display problem for images in table

### PR DESCRIPTION
**Pull request by peterleimbach**

### CODE REVIEW

**Comment by peterleimbach** [comment]
fixes #1091

File: docs/EN/Configuration/OmnipodEros.rst [hunk 1]
```diff
@@ -480,63 +488,78 @@ Pod Management Menu
...
-    * - |pulse_log|
-      - **Read pulse log:** 
-    
-    	Sends the active pod pulse log to the clipboard		    
+
+* **Read pulse log:**
+
+    |pulse_log|
+
+    	Sends the active pod pulse log to the clipboard
```
Comment (jonaschn) [inline docs/EN/Configuration/OmnipodEros.rst:562]: This line is indented one level too far

File: docs/EN/Configuration/OmnipodEros.rst [hunk 2]
```diff
@@ -441,37 +441,45 @@ Fields
...
+* **SET TIME:**
+
+    |set_time|
+
+    When pressed this will update the time on the pod with the current time on your phone.
+
+* **SUSPEND:**
+
+    |suspend|
+    Suspends the active pod
```
Comment (jonaschn) [inline docs/EN/Configuration/OmnipodEros.rst:478]: Missing newline (otherwise inconsistent)

File: docs/EN/Configuration/OmnipodEros.rst [hunk 3]
```diff
@@ -441,37 +441,45 @@ Fields
...
+
+* **RESUME DELIVERY:**
+
+    |resume|
 	Resumes the currently suspended, active pod
```
Comment (jonaschn) [inline docs/EN/Configuration/OmnipodEros.rst:483]: Missing newline (otherwise inconsistent)

Comment (jonaschn) [reply docs/EN/Configuration/OmnipodEros.rst:483]: This line currently renders as indented too.
image
(see [here](https://androidaps.readthedocs.io/en/latest/Configuration/OmnipodEros.html#icons))

**Review by jonaschn — COMMENTED** [review]
@peterleimbach @xJoe Thanks for your fix. I've spotted some inconsistencies that should be fixed before checking Crowdin.

**Pull request merged**